lfs/transfer_queue.go

```diff
@@ -76,6 +83,26 @@ func (q *TransferQueue) Wait() {
 	}
 
 	q.wait.Wait()
+
+	// Handle any retries
+	close(q.retriesc)
+	q.retrywait.Wait()
+	atomic.StoreUint32(&q.retrying, 1)
+
+	if len(q.retries) > 0 {
+		tracerx.Printf("tq: retrying %d failed transfers", len(q.retries))
+		if q.batcher != nil {
+			q.batcher.Reset()
+		}
+		for _, t := range q.retries {
+			q.Add(t)
+		}
+		if q.batcher != nil {
+			q.batcher.Exit()
+		}
+		q.wait.Wait()
+	}
+
```

Comment:
It won't have any effect here, since the TQ isn't meant to be reused, but let's add it to pave the way for a reusable TQ in the future.